src/components/Video/index.tsx

```diff
@@ -21,10 +22,22 @@ const Video: React.FC<VideoProps> = (props) => {
     env,
     autoplay = false,
     playthrough = false,
-    // default 16:9
-    aspectRatio = 0.5625,
+    viewportPercentage = 75,
+    shrinkToFit = false,
   } = props;
   const muted = autoplay;
+  const containerRef = useRef();
+  const [aspectRatio, setAspectRatio] = useState(9 / 16); // default 16:9
```

Comment:
Same as comment below regarding the naming and comment

Reply:
clarified